src/test/java/cloud/eppo/api/EppoValueTest.java

```diff
@@ -0,0 +1,63 @@
+package cloud.eppo.api;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class EppoValueTest {
```

Comment:
We actually have an `EppoValueTest` already, but it was left in `src/test/java/cloud/eppo/` (vs. API).
How about moving it here and then merging in the below test cases?

Reply:
✔️